test/integration/search_test.rb

```diff
@@ -50,11 +50,16 @@ class SearchTest < SystemTest
     end
   end
 
-  setup { Rubygem.per_page = 2 }
+  setup do
+    3.times do |i|
+      rubygem = create(:rubygem, name: "ruby#{i}", number: "1.#{i}.0")
```

Comment:
`number` creates corresponding version. Our query condition has [`versions.indexed`](https://github.com/spk/rubygems.org/blob/c346b910db404bad1ff209266e4ebd740a5a22b0/app/models/concerns/rubygem_searchable.rb#L127) and without version response will always be empty.

I would suggest not to use `number: "1.#{i}.0"` and let it be `number: "1.0.0"`. Someone may try to read into need of *different* version number when there is none.